src/api/public/apidocs-new/paths/published_project_name_repository_name.yaml

```diff
@@ -0,0 +1,35 @@
+get:
+  summary: List the architectures of the pair project/repository with published binaries.
+  description: >
+    Get a list of the architectures of the given pair project and repository that already have published binaries.
+  security:
+    - basic_authentication: []
+  parameters:
```

Comment:
@saraycp there is also the `view=status` parameter for this endpoint

Reply:
I'm going to add that endpoint in another PR

Reply:
ahh right, returns a different schema, fine then